apiserver/common/leadership.go

```diff
@@ -44,16 +44,18 @@ func (s leadershipPinningBackend) Machine(name string) (LeadershipMachine, error
 	return leadershipMachine{m}, nil
 }
 
-// API exposes leadership pinning and unpinning functionality for remote use.
+// LeadershipPinningAPI exposes leadership pinning and unpinning functionality
+// for remote use.
 type LeadershipPinningAPI interface {
```

Comment:
I don't think this is actually used anywhere now that you've re-declared it on-site for the upgrade-series facade.

Might as well ditch it.